crates/rolldown/src/bundler.rs

```diff
@@ -120,6 +123,8 @@ impl Bundler {
       .build_end(error_for_build_end_hook.map(|error| HookBuildEndArgs { error }).as_ref())
       .await?;
 
+    scan_stage_output.warnings.append(&mut self.warnings);
```

Comment:
You should append these warnings in the end of `write`/`generate`. It's more predictable.

Reply:
I'm not sure whether this should be added here, because if we are simply using the `scan` exposed by `napi`, this warning won't actually be exposed.

https://github.com/rolldown/rolldown/blob/a64f4825e7359dc5474a8e2fc78ce4ef144561d3/crates/rolldown_binding/src/bundler.rs#L115-L119

Reply:
👌Get it